bsoid_app.py

```diff
@@ -61,21 +57,19 @@ def get_env_variable(var_name, default_value=None):
 [features, _] = load_feats(WORKING_DIR, PREFIX)
 [sampled_features, _] = load_embeddings(WORKING_DIR, PREFIX)
 [_, assignments, _, _] = load_clusters(WORKING_DIR, PREFIX)
-learning_protocol = machine_learner.protocol(WORKING_DIR, PREFIX, features, sampled_features, assignments)
+learning_protocol = machine_learner.Protocol(WORKING_DIR, PREFIX, features, sampled_features, assignments)
```

Comment:
where does this capitalization come from?

Reply:
this is something Mert had suggested, Protocol is a class, and I guess classes are usually capitalized in python